modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -425,11 +515,12 @@ PERF_TEST_P_(DivCPerfTest, TestPerformance)
 
 PERF_TEST_P_(DivRCPerfTest, TestPerformance)
 {
-    compare_f cmpF = get<0>(GetParam());
-    Size sz = get<1>(GetParam());
-    MatType type = get<2>(GetParam());
-    int dtype = get<3>(GetParam());
-    cv::GCompileArgs compile_args = get<4>(GetParam());
+    compare_f cmpF;
+    cv::Size sz;
+    MatType type = -1;
+    int dtype = -1;
+    cv::GCompileArgs compile_args;
+    std::tie(cmpF, sz, type, dtype, compile_args) = GetParam();
```

Comment:
The same as above. Could you please add `double scale` value and add its initialization via test configuration parameter?

Reply:
Applied